js/cex.js

```diff
@@ -837,57 +842,59 @@ module.exports = class cex extends Exchange {
             const quoteId = this.safeString (order, 'symbol2');
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
-            symbol = base + '/' + quote;
+            if (base !== undefined && quote !== undefined) {
```

Comment:
we need to use brackets here, like so `((base !== undefined) && (quote !== undefined)) {